From: Stefano Zacchiroli <zack@debian.org>
Date: Tue, 19 May 2009 17:25:59 +0200
Subject: [PATCH] Add to ocamldoc support for native code dynamic loading

That enables using ocamldoc flags such as "-g" also with
ocamldoc.opt. Patch is taken from OCaml CVS and is already
integrated upstream for future releases.
---
 man/ocamldoc.m |    5 +----
 1 files changed, 1 insertions(+), 4 deletions(-)

diff --git a/man/ocamldoc.m b/man/ocamldoc.m
index 5d1ed6a..757ec58 100644
--- a/man/ocamldoc.m
+++ b/man/ocamldoc.m
@@ -110,10 +110,7 @@ to display it.
 .TP
 .BI \-g \ file
 Dynamically load the given file (which extension usually is .cmo or .cma),
-which defines a custom documentation generator. This option is supported by the
-.BR ocamldoc (1)
-command, but not by its native-code version
-.BR ocamldoc.opt .
+which defines a custom documentation generator.
 If the given file is a simple one and does not exist in
 the current directory, then ocamldoc looks for it in the custom
 generators default directory, and in the directories specified with the
-- 
1.6.2.4

